bin/flutter

```diff
@@ -175,14 +175,14 @@ fi
 set +e
 # FLUTTER_TOOL_ARGS isn't quoted below, because it is meant to be considered as
 # separate space-separated args.
-"$DART" $FLUTTER_TOOL_ARGS "$SNAPSHOT_PATH" "$@"
+# "$DART" $FLUTTER_TOOL_ARGS "$SNAPSHOT_PATH" "$@"
```

Comment:
These changes should probably be reverted?

Reply:
Yep, the flutter tool was using the snapshot even with source changes.